Azure/azure-key-vault/ingest/parser.yml

```diff
@@ -0,0 +1,38 @@
+name: azure-key-vault
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          event.category: ["database"]
+          event.dataset: "keyvault"
+          event.outcome: "success"
+          event.action: "{{json_event.message.operationName}}"
+          event.type: ["access"]
+          cloud.provider: "azure"
+          cloud.service.name: "Azure Key Vault"
+          "@timestamp": "{{json_event.message.time}}"
+          source.ip: "{{json_event.message.callerIpAddress}}"
+          user.name: "{{json_event.message.identity.claim.unique_name}}"
+          user_agent.original: "{{json_event.message.properties.clientInfo}}"
+          url.original: "{{json_event.message.properties.requestUri}}"
+          tls.version: "{{json_event.message.properties.tlsVersion}}"
+          azure.key_vault.result.type: "{{json_event.message.resultType}}"
+          azure.key_vault.result.description: "{{json_event.message.resultDescription}}"
+          azure.key_vault.result.signature: "{{json_event.message.resultSignature}}"
+          azure.key_vault.correlation_id: "{{json_event.message.correlationId}}"
+          azure.key_vault.resource_id: "{{json_event.message.resourceId}}"
+          azure.key_vault.identity.claim.appid: "{{json_event.message.identity.claim.appid}}"
+          azure.key_vault.properties.tenantid: "{{json_event.message.properties.properties.tenantId}}"
+          azure.key_vault.properties.sku.family: "{{json_event.message.properties.properties.sku.Family}}"
+          azure.key_vault.properties.sku.name: "{{json_event.message.properties.properties.sku.Name}}"
+          azure.key_vault.properties.sku.capacity: "{{json_event.message.properties.properties.sku.Capacity}}"
+
+      - set:
+          event.outcome: "failure"
```

Comment:
We should also add "denied" to event.type

Reply:
I agree

Reply:
I did answer in DM, but will duplicate here
Seems like in ECS we do not have `denied` for `database` and for `outcome`
https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-outcome.html
https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-category.html#ecs-event-category-database